src/AddTodoForm.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const AddTodoForm = (props) => {
+
+    function handleAddTodo(event) {
+      event.preventDefault(); 
+      const form = event.target;
+      const todoTitle = form["title"].value; 
```

Comment:
Nitpick (doesn't require changes):
You can use dot notation for `form.title.value` instead of bracket notation `form["title"].value`. You'll have to use bracket notation, if... for example:
```html
<input id="todoTitle" name="todo title" type="text" />
```
the `name="todo title"` has a space within the quotes. Then, you'll need to use `form["todo title"].value` since `form.todo title.value` is not valid JavaScript.

Reply:
Thank you for your adice , I`ll keep it in mind.